src/components/TaskCard/index.jsx

```diff
@@ -1,20 +1,42 @@
 import PropTypes from "prop-types";
 import { useDispatch } from "react-redux";
-import { deleteTask } from "/src/store/actions";
-import { ICON_DELETE } from "/src/utils/constants/icons";
+import { deleteTask, toggleTaskDone } from "/src/store/actions";
+import { ICON_DELETE, ICON_DONE } from "/src/utils/constants/icons";
 import { formatDate } from "/src/utils/helpers/formatDate";
 import "./index.scss"
 
 const TaskCard = ({ task }) => {
   const dispatch = useDispatch();
 
+  function getDaysToCompleteTask(startDate, endDate) {
+    const millisecondsInADay = 24 * 60 * 60 * 1000;
+    const diffInMilliseconds = endDate - startDate;
+    const diffInDays = Math.floor(diffInMilliseconds / millisecondsInADay) + 1;
+
+    const daysStr = (diffInDays > 1) ? `${diffInDays} days` : `${diffInDays} day`;
+
+    return daysStr;
+  }
+
   const handleDelete = () => {
     dispatch(deleteTask(task.id));
   };
+
+  const handleToggleDone = () => {
+    dispatch(toggleTaskDone(task.id));
+  };
+
   return (
     <div className="task-card">
-      <p className="task-card__title">{task.title}</p>
+      <p className="task-card__title" style={{ textDecoration: task.done ? 'line-through' : 'none' }}>{task.title}</p>
```

Comment:
can we put this style in class? and also can we use [classname](https://www.npmjs.com/package/classnames) ?